src/video_core/engines/kepler_memory.h

```diff
@@ -63,6 +68,18 @@ class KeplerMemory final {
                         return static_cast<GPUVAddr>((static_cast<GPUVAddr>(address_high) << 32) |
                                                      address_low);
                     }
+
+                    u32 BlockWidth() const {
+                        return 1 << block_width.Value();
```

Comment:
```suggestion
                        return 1U << block_width.Value();
```

otherwise this causes a sign-conversion. Ditto for the other two below.